internal/secrets/localsecrets/localsecrets.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package localsecrets provides a way to encrypt and decrypt small messages
+// without making network calls to a third party service.
+package localsecrets
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/base64"
```

Comment:
Nit: usually imports are grouped into base library stuff, then a blank line, then other stuff. (so the github.com and golang.org ones would be in a block below).

@jba is this documented somewhere? Would be good to add to coding conventions.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#imports (and noted in other places too)